src/test/resources/product.csv

```diff
@@ -0,0 +1,26 @@
+id,brand,device,price

```

Comment:
is this file used in your repo? if not, can we have at least one of the filter Apple / Samsung test cases use it for assertions.